nextcord/channel.py

```diff
@@ -716,6 +716,7 @@ async def create_thread(
         message: Optional[Snowflake] = None,
         auto_archive_duration: ThreadArchiveDuration = MISSING,
         type: Optional[ChannelType] = None,
+        invitable: bool = True,
```

Comment:
Ideally this should default to `None`, which means "default to Discord's default" (by not adding unnecessary fields to the payload)